compiler/src/dotty/tools/dotc/core/Contexts.scala

```diff
@@ -95,7 +95,7 @@ object Contexts {
   inline def atPhaseNoEarlier[T](limit: Phase)(inline op: Context ?=> T)(using Context): T =
     op(using if !limit.exists || limit <= ctx.phase then ctx else ctx.withPhase(limit))
 
-  inline private def inMode[T](mode: Mode)(inline op: Context ?=> T)(using ctx: Context): T =
+  inline def inMode[T](mode: Mode)(inline op: Context ?=> T)(using ctx: Context): T =
```

Comment:
We should find a more obtuse name for this, as I accidentally used it when I meant to use withMode.